vpr/src/route/connection_router.cpp

```diff
@@ -531,9 +623,22 @@ void ConnectionRouter<Heap>::timing_driven_add_to_heap(const t_conn_cost_params
         next_ptr->set_prev_edge(from_edge);
         next_ptr->set_prev_node(from_node);
 
+        if (run_rcv && current->path_data) {
+            next_ptr->path_data->backward_cong = next.path_data->backward_cong;
```

Comment:
So rather than copying the data, this could just move path_data pointer/StrongId from next_ptr, and then you don't need to double allocate.

Reply:
Not a bad idea, but I would have to be careful with the `free` function. This is a good optimizing change.

Reply:
This should likely be `rcv_path_manager.move(next_ptr, &next)`?